Sources/Core/AccessTokenWallet.swift

```diff
@@ -26,6 +26,18 @@ class AccessTokenWallet {
   let graphConnectionProvider: GraphConnectionProviding
   let graphRequestPiggybackManager: GraphRequestPiggybackManaging.Type
 
+  // If we do not disable explicit_type_interface and also add the type we get a
+  // redundant_type_annotation warning. We cannot have both of these rules enabled
+  // in this use case.
+  // swiftlint:disable explicit_type_interface
```

Comment:
`explicit_type_interface` has a configuration of `allow_redundancy`.
```yml
explicit_type_interface:
  allow_redundancy: true
  excluded:
    - local
```

Reply:
👍